test/functional/page_objects/dashboard_page.ts

```diff
@@ -588,7 +588,9 @@ export class DashboardPageObject extends FtrService {
 
   public async getPanelTitles() {
     this.log.debug('in getPanelTitles');
-    const titleObjects = await this.find.allByCssSelector('span.embPanel__titleInner');
+    const titleObjects = await this.find.allByCssSelector(
+      '[data-test-subj=embeddablePanelTitleInner] .embPanel__titleText'
+    );
```

Comment:
Now that the saved object description is being used, the titles for the panels used in the `panel_clone` and `dashboard_clone` had extra white space when fetched - we no longer have to worry about this if we modify how we grab the panel title to **only** include the panel title and not the description icon.

The `panel_clone` and `dashboard_clone` tests **should** have started failing as part of the https://github.com/elastic/kibana/pull/178139, but they didn't because we weren't setting the default panel description for the saved objects used in these tests.